daemon/daemon.go

```diff
@@ -760,9 +760,23 @@ func (daemon *Daemon) Shutdown() error {
 
 				go func() {
 					defer group.Done()
-					// If container failed to exit in 10 seconds of SIGTERM, then using the force
-					if err := c.Stop(10); err != nil {
-						logrus.Errorf("Stop container %s with error: %v", c.ID, err)
+					if c.IsPaused() {
```

Comment:
Yup, that sounds right. This is an area I already have marked for a subsequent PR to improve on WIndows, but your suggestion sounds the best.